mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,9 @@
       android:label="@string/app_name"
       android:icon="@mipmap/ic_launcher"
       android:allowBackup="false"
-      android:theme="@style/AppTheme">
+      android:theme="@style/AppTheme"
+      android:usesCleartextTraffic="true">
```

Comment:
Not sure that is needed for us

Reply:
Okay, I removed that and just changed the links to the meditations)